lms/djangoapps/discussion/tests/test_signals.py

```diff
@@ -63,7 +72,8 @@ def test_comment_created_signal_msg_not_sent_with_site_config_disabled(
         site_config.site_values = enable_notifications_cfg
         site_config.save()
         mock_get_current_site.return_value = self.site
-        signals.comment_created.send(sender=self.sender, user=self.user, post=self.post)
+        with mock.patch('lms.djangoapps.discussion.rest_api.tasks.send_response_notifications.apply_async'):
+            signals.comment_created.send(sender=self.sender, user=self.user, post=self.post)
```

Comment:
can we have some assertion check here to see if the signal was called correctly?